### PR TITLE
vite

### PR DESCRIPTION
**Pull request by gregberge**

### CODE REVIEW

File: apps/frontend/package.json [hunk 1]
```diff
@@ -28,12 +28,11 @@
...
     "@types/react-dom": "^18.2.23",
     "@types/react-helmet": "^6.1.11",
+    "@vitejs/plugin-react": "^4.2.1",
     "ariakit": "^2.0.0-next.44",
-    "assets-webpack-plugin": "^7.1.1",
     "autoprefixer": "^10.4.19",
     "axios": "^1.6.8",
     "clsx": "^2.1.0",
-    "core-js": "^3.36.1",
     "css-loader": "^6.10.0",
```
Comment (wojtekmaj) [inline apps/frontend/package.json:36]: Still needed?

File: apps/frontend/vite.config.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      },
+    },
+    resolve: {
+      alias: [
+        {
+          find: "@",
+          replacement: fileURLToPath(new URL("./src", import.meta.url)),
+        },
+      ],
+    },
```
Comment (wojtekmaj) [inline apps/frontend/vite.config.ts:31]: Should not be needed with package.json imports 🤔

File: apps/frontend/vite.config.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    define: {
+      "process.env.NODE_ENV": JSON.stringify(mode),
+    },
```
Comment (wojtekmaj) [inline apps/frontend/vite.config.ts:34]: Have a read: https://vitejs.dev/guide/env-and-mode 
Might be an interesting alternative to Node.js-mocking-era `process.env.*`.

File: apps/frontend/package.json (hunk 1 above)
Comment (gregberge) [reply apps/frontend/package.json:36]: Removed in another release thanks to [Knip](https://knip.dev/).

File: apps/frontend/vite.config.ts (hunk 2 above)
Comment (gregberge) [reply apps/frontend/vite.config.ts:31]: I will try it!

File: apps/frontend/vite.config.ts (hunk 3 above)
Comment (gregberge) [reply apps/frontend/vite.config.ts:34]: Lot of libraries still relies on process.env.NODE_ENV, I think it's better to stick on it for now. In my code not necessarily but for dependencies.

**Pull request merged**